ansible/playbooks/start-redpanda.yml

```diff
@@ -61,3 +61,32 @@
         --api-urls {{ groups["redpanda"] | map('extract', hostvars) | map(attribute='private_ip') | product([':9644']) | map('join') | join(',') }}
       {% endif %}
     when: rack is defined and rack != -1
+
+  - name: enable shadow indexing
+    notify:
+      - restart redpanda-tuner
+      - restart redpanda
+    vars:
+      use_public_ips: "{{ advertise_public_ips | d() | bool }}"
+      this_node: "{{ inventory_hostname if use_public_ips else private_ip }}"
+    shell: |
+      sudo -u redpanda rpk cluster config set cloud_storage_bucket {{ si_bucket_name }} --api-urls {{ "%s:%d" % (this_node, 9644) }}
```

Comment:
I think we can add a var for api_urls that contains all the admin API endpoints, similar to what you did [here](https://github.com/redpanda-data/deployment-automation/commit/8579fdde423714696881c07dc1ee331652c31ffd#diff-c28b7b45769b3c8b11796f623bc3b3aa49a28fcd79d5f0cbb2d6ebc757994e8aR58)